packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/TokenBalance.tsx

```diff
@@ -0,0 +1,51 @@
+import { BigNumber } from 'ethers'
+
+import { ERC20BridgeToken } from '../../../hooks/arbTokenBridge.types'
+import { NativeCurrencyErc20 } from '../../../hooks/useNativeCurrency'
+import { Loader } from '../../common/atoms/Loader'
+import { TokenSymbolWithExplorerLink } from '../../common/TokenSymbolWithExplorerLink'
+import { formatAmount } from '../../../util/NumberUtils'
+
+import { NetworkType } from './utils'
+
+export function TokenBalance({
+  forToken,
+  balance,
+  on,
+  prefix = '',
+  tokenSymbolOverride
+}: {
+  forToken: ERC20BridgeToken | NativeCurrencyErc20 | null
+  balance: BigNumber | null
+  on: NetworkType
+  prefix?: string
+  tokenSymbolOverride?: string
+}) {
+  const isParentChain = on === NetworkType.parentChain
+
+  if (!forToken) {
+    return null
+  }
+
+  if (!balance) {
+    return <Loader color="white" size="small" />
```

Comment:
this is the only change

I used this in favour of `StyledLoader` because that component only returns this line

might as well kill that function later